contrib/pkcs11-helper-man

```diff
@@ -0,0 +1 @@
+pkcs11-helper
```

Comment:
things without a @subpackage in the template don't need a symlink on disk (the ci fails cause of that)